pkg/restapi/operation/didexchange/didexchange.go

```diff
@@ -217,10 +210,26 @@ func (c *Operation) QueryConnections(rw http.ResponseWriter, req *http.Request)
 		},
 	}
 
-	err = json.NewEncoder(rw).Encode(response)
+	c.writeResponse(rw, response)
+}
+
+// getQueryParams unmarshals query strings values to interface value
+func (c *Operation) getQueryParams(v interface{}, vals url.Values) error {
```

Comment:
I don't see any usage of c within this function.

Reply:
made it private static